Strings/check_if_strings_are_rotation_of_eachother.cpp

```diff
@@ -0,0 +1,37 @@
+/***************************************************************************
+
+A Program to check if strings are rotations of each other or not.
+
+***************************************************************************/
+
+SOLUTION (in C++):
```

Comment:
Comment this line and also give explanation in multi-line comments.

Reply:
Should I make a pull request again after I have done the necessary changes or is there any way to do the changes here itsefl?

Reply:
Just the commit and push the changes through the same branch, it will reflect here.

Reply:
I have done it, just check it once.

Reply:
The comments are good, just also comment the line "Solution in C++" only the code should remain uncommented.

Reply:
Done.